install/kubernetes/templates/helm/istio/values.yaml.tmpl

```diff
@@ -18,6 +18,13 @@ global:
   hyperkube:
     repository: {HYPERKUBE_HUB}
     tag: {HYPERKUBE_TAG}
+  # addons
+  grafana:
+    repository: {MIXER_HUB}/grafana
+    tag: {MIXER_TAG}
+  servicegraph:
```

Comment:
also unclear if MIXER_TAG should be used here.

Reply:
same as before for global vars, for the tag, I copied that from `istio-ingress.yaml.tmpl`. what tag should go there then?

Reply:
@luisyonaldo your correct, this is located in https://github.com/istio/istio/blob/master/install/kubernetes/templates/addons/grafana.yaml.tmpl#L30 and contains MIXER_HUB and MIXER_TAG.  I wouldn't have expected that but it does make sense (since the addons integrate with mixer).